woocommerce/woocommerce-gutenberg-products-block
# Add a stock filter block powered by the interactivity API

## What

This PR:

* Adds an interactivity API powered dropdown component that is in theory reusable across blocks.
* Adds the MVP stock filter with a checkbox list and dropdown mode to filter stock by stock statuses

## Testing Instructions

* Create a page or post with the Product Collection Beta Block,  new Collection FIlters block and this Stock Filter block as a child of Collection Filters
* Preview the block and see that checking different statuses changes which products are displayed
* Adjust block attributes to show the dropdown (`displayStyle` of `dropdown`). It currently only supports single select, choose a stock status and see the filtering happens appropriately.
